core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapCompareRequest.java

```diff
@@ -16,44 +16,48 @@
 package com.b2international.snowowl.core.request;
 
 import java.util.List;
-import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import javax.validation.constraints.Min;
 
 import com.b2international.snowowl.core.codesystem.CodeSystemRequests;
+import com.b2international.snowowl.core.compare.ConceptMapCompareConfigurationProperties;
 import com.b2international.snowowl.core.compare.ConceptMapCompareResult;
+import com.b2international.snowowl.core.compare.MapCompareSourceAndTargetEquivalence;
 import com.b2international.snowowl.core.domain.BranchContext;
 import com.b2international.snowowl.core.domain.ConceptMapMapping;
 import com.b2international.snowowl.core.domain.ConceptMapMappings;
 import com.b2international.snowowl.core.uri.ComponentURI;
+import com.google.common.base.Equivalence.Wrapper;
 import com.google.common.collect.ArrayListMultimap;
-import com.google.common.collect.HashMultimap;
 import com.google.common.collect.ListMultimap;
 import com.google.common.collect.Lists;
-import com.google.common.collect.Multimap;
 import com.google.common.collect.Sets;
 import com.google.common.collect.Sets.SetView;
 
 /**
 * @since 7.8
 */
 public final class ConceptMapCompareRequest extends ResourceRequest<BranchContext, ConceptMapCompareResult> {
+	public static final int DEFAULT_MEMBER_COMPARE_LIMIT = 50_000;
 	
 	private static final long serialVersionUID = 1L;
 	
 	private static final int DEFAULT_MEMBER_SCROLL_LIMIT = 10_000;
 	
 	private final ComponentURI baseConceptMapURI;
 	private final ComponentURI compareConceptMapURI;
+	private final Set<ConceptMapCompareConfigurationProperties> selectedConfig;
 	
 	@Min(0)
 	private int limit;
```

Comment:
Are we using this now? I'm not seeing any statement nor filter that uses this value.

Reply:
It is used when creating the `MapCompareSourceAndTargetEquivalence` instance and in the `isSourceEqual()` method.